packages/wonder-blocks-banner/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@khanacademy/wonder-blocks-banner",
-  "version": "4.1.16",
+  "version": "4.1.17",
```

Comment:
Could you please revert this change? `pnpm changeset` should take care of this for us as soon as these changes are released to npm.